kafka/conn.py

```diff
@@ -301,7 +308,7 @@ def __init__(self, host, port, afi, **configs):
         if self.config['ssl_context'] is not None:
             self._ssl_context = self.config['ssl_context']
         self._sasl_auth_future = None
-        self.last_attempt = 0
```

Comment:
Yep!

Reply:
Okay, I agree, that's a valid concern.

I don't think renaming the variable back is a good idea, as it is now updated more often than before and cannot be used to indicate the same things.

I can reinstate `last_activity` and make it be updated in the same places as earlier. Only downside here is that it would be a variable that is never read internally in the library, making it excess, but it would avoid breaking compatibility. Are you okay with this solution?

Reply:
Almost, let me make a small adjustment. There are some points where I'd like to update both values via `self.last_activity = self.last_attempt = time.time()`.

Reply:
Ok for me! 

Just remember that the value of keeping `self.last_activity` is diminishing if the variable isn't updated at the same places in time as before this PR. I don't know where you want to make the changes, so I can't really comment more on it, though.

Reply:
@wbarnha Happy to resolve this conversation?